gsy_framework/enums.py

```diff
@@ -25,6 +25,7 @@ class CoefficientAlgorithm(Enum):
     STATIC = 1
     DYNAMIC = 2
     NO_COMMUNITY_SELF_CONSUMPTION = 3
+    LIVE_STREAM = 4
```

Comment:
This will be implemented in the frame of GSYE-672, right?

Reply:
Exactly. This is just a placeholder for the following ticket